framework/auth/views.py

```diff
@@ -944,7 +944,7 @@ def register_user(**kwargs):
         )
 
     if settings.CONFIRM_REGISTRATIONS_BY_EMAIL:
-        send_confirm_email_async(user, email=user.username)
+        send_confirm_email(user, email=user.username)
```

Comment:
This line was previously causing the default email template to be sent, rather than the campaign-specific one.